.github/workflows/locize-pull-published-sync-pr.yml

```
@@ -0,0 +1,53 @@
+          body: |
+            This PR updates translations from locize.
+            - Version: `${{ github.event.client_payload.payload.version }}`
+            - Automated update from GitHub Actions
+          labels: "translations, automated"
+          draft: false
+
+      - name: Check PR Outputs
+        if: steps.create_pr.outputs.pull-request-number
+        run: |
+          echo "Pull Request Number - ${{ steps.create_pr.outputs.pull-request-number }}"
+          echo "Pull Request URL - ${{ steps.create_pr.outputs.pull-request-url }}"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)